packages/table/src/Cell.tsx

```diff
@@ -5,30 +5,58 @@ import { commonCellStyles } from './styles';
 
 export const tdInnerDiv = createDataProp('td-inner-div');
 
-type CellProps = HTMLElementProps<'td', HTMLTableCellElement>;
+interface HeaderCellProps
+  extends HTMLElementProps<'th', HTMLTableHeaderCellElement> {
+  isHeader?: true;
+}
 
-const tdStyles = css`
+interface TableCellProps extends HTMLElementProps<'td', HTMLTableCellElement> {
+  isHeader?: false;
+}
+
+type CellProps = HeaderCellProps | TableCellProps;
+
+const baseStyles = css`
   line-height: 20px;
   position: relative;
 `;
 
+const thStyles = css`
+  font-weight: bold;
+`;
+
 const innerDivStyles = css`
   display: flex;
   align-items: center;
 `;
 
 const Cell = React.forwardRef(
-  ({ children, className, ...rest }: CellProps, ref: React.Ref<any>) => {
+  (
+    { children, className, isHeader = false, ...rest }: CellProps,
+    ref: React.Ref<any>,
+  ) => {
+    const Element = isHeader ? 'th' : 'td';
+
+    const props: Partial<CellProps> = {
+      ref,
+      className: cx(
+        commonCellStyles,
+        baseStyles,
+        { [thStyles]: isHeader },
+        className,
+      ),
+    };
+
+    if (isHeader) {
+      props.scope = 'row';
+    }
+
     return (
-      <td
-        ref={ref}
-        className={cx(commonCellStyles, tdStyles, className)}
-        {...rest}
-      >
+      <Element {...props} {...rest}>
```

Comment:
nit: Would prefer that we call this Root or Component to be consistent with other occasions across the repo where we do something similar, but this isn't imperative

Reply:
Done